more_autograding_examples/cpp_simple_lab/submissions/julian_compile_error.cpp

```diff
@@ -18,7 +18,7 @@ const int DaysInMonth[13] = { 0, 31, 28, 31, 30, 31, 30,
 bool 
 is_leap_year( int year )
 {
-  return year % 4 == 0 && ( yera % 100 != 0 || year % 400 == 0 );
```

Comment:
I think this may be part of the example. Instead of having a spelling error, how about having it say "years". This will still cause a compile error, but be spelled correctly.

Reply:
This test case was deliberate and should remain as-is so it is clear how it is wrong if anyone stumbles upon it in the future.

Reply:
Do you think a comment should be added to the file to help clarify the error?